packages/verified-fetch/src/utils/get-resolved-accept-header.ts

```diff
@@ -0,0 +1,42 @@
+import { isExplicitAcceptHeader, isExplicitFormatQuery, isExplicitIpldAcceptRequest } from './is-accept-explicit.js'
+import { queryFormatToAcceptHeader } from './select-output-type.js'
+import type { ParsedUrlStringResults } from './parse-url-string.js'
+import type { ComponentLogger } from '@libp2p/interface'
+
+export interface ResolvedAcceptHeaderOptions {
+  query?: ParsedUrlStringResults['query']
+  headers?: RequestInit['headers']
+  logger?: ComponentLogger
```

Comment:
`helia.logger` will always be set so I don't think this field needs to be optional.